tests/misaligned.test.js

```diff
@@ -0,0 +1,27 @@
+import { generateColorCombinations } from "../misaligned.js";
+import { expect } from "chai";
+
+let colorCombinationsResult = generateColorCombinations();
+
+// Capture console.log output
+let loggedOutput = [];
+console.log = (output) => loggedOutput.push(output);
+
+colorCombinationsResult.forEach((colorCombination) =>
+  console.log(colorCombination)
+);
+
+expect(loggedOutput.length).equals(25);
+
+// Test for misalignments
+loggedOutput.forEach((colorCombination) => {
+  let colorParts = colorCombination.split("|");
+
+  expect(colorParts[0].length).equals(4); // This will fail if the numeric value is not aligned with the separator
```

Comment:
The expected numbers (4, 7, ) will need to change with the implementation
e.g., if we fix the alignment in the code (not now but in the next assignment), add more spacing while retaining the alignment across the rows, this test will fail while the alignment would be ok.
Try testing the alignment (=whether the `|` symbols align at the same position in all the rows) instead of fixing on the spacing?